### PR TITLE
C-ICAP: fix templating in virus_scan.conf

### PR DESCRIPTION
**Pull request by mimugmail**
GH tells me it's unable to merge, perhaps you can have a quick look @fichtner 
Thanks!

### CODE REVIEW

**Comment by fichtner** [comment]
it changes cmaxobjectsize to maxobjectsize again. I can fix this up later, just leave it open.

File: www/c-icap/src/opnsense/service/templates/OPNsense/CICAP/virus_scan.conf [hunk 1]
```diff
@@ -8,19 +8,19 @@ virus_scan.ScanFileTypes {{ OPNsense.cicap.antivirus.scanfiletypes.replace(',',
...
-virus_scan.SendPercentData 5
+virus_scan.SendPercentData {{ OPNsense.cicap.antivirus.sendpercentdata }}
 {% endif %}
 {% if helpers.exists('OPNsense.cicap.antivirus.startsendpercentdataafter') and OPNsense.cicap.antivirus.startsendpercentdataafter != '' %}
-virus_scan.StartSendPercentDataAfter 2M
+virus_scan.StartSendPercentDataAfter {{ OPNsense.cicap.antivirus.startsendpercentdataafter }}
 {% endif %}
 {% if helpers.exists('OPNsense.cicap.antivirus.allow204responses') and OPNsense.cicap.antivirus.allow204responses != '' %}
-virus_scan.Allow204Responces on
+virus_scan.Allow204Responces {{ OPNsense.cicap.antivirus.allow204responses }}
```
Comment (fabianfrz) [inline www/c-icap/src/opnsense/service/templates/OPNsense/CICAP/virus_scan.conf]: @mimugmail please be careful - on/off switches of checkboxes are 1/0 - you may break the config here

File: www/c-icap/src/opnsense/service/templates/OPNsense/CICAP/virus_scan.conf [hunk 2]
```diff
@@ -8,19 +8,19 @@ virus_scan.ScanFileTypes {{ OPNsense.cicap.antivirus.scanfiletypes.replace(',',
...
 {% endif %}
 {% if helpers.exists('OPNsense.cicap.antivirus.passonerror') and OPNsense.cicap.antivirus.passonerror != '' %}
-virus_scan.PassOnError off
+virus_scan.PassOnError {{ OPNsense.cicap.antivirus.passonerror }}
```
Comment (fabianfrz) [inline www/c-icap/src/opnsense/service/templates/OPNsense/CICAP/virus_scan.conf]: same here

**Comment by fabianfrz** [comment]
@fichtner I fixed the merge conflict on GitHub.
@mimugmail you need to pull if you work locally.

**Comment by mimugmail** [comment]
Sorry Jungs fuer den Mehraufwand, zur Zeit is bei mir der Wurm drin. Jetzt sollts passen.

**Comment by fichtner** [comment]
Immer mit der Ruhe, alles prima. Danke dafür 👍

**Pull request merged**